pages/api/v1/user-created-bounties/submissions/updateSubmissionStatus.ts

```
@@ -0,0 +1,98 @@
+
+		if (![EUserCreatedBountySubmissionStatus.APPROVED, EUserCreatedBountySubmissionStatus.REJECTED, EUserCreatedBountySubmissionStatus.PAID].includes(updatedStatus)) {
+			return res.status(400).json({ message: 'Invalid Updated Status Param' });
+		}
+		if (!parentBountyProposerAddress?.length || !encodedParentBountyProposerAddress) {
+			return res.status(400).json({ message: 'Invalid Proposer Address' });
+		}
+		if (!submissionProposerAddress?.length || !encodedSubmissionProposerAddress) {
+			return res.status(400).json({ message: 'Invalid Proposer Address' });
+		}
+
+		if (isNaN(parentBountyIndex)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)